### DISCUSSION TITLE
Using WSL - test scenes not working

### DISCUSSION THREAD

**Discussion by KneeShe-2004** [empriselab/RCareWorld#98] [Help - Setting up / Configuring] [answered]
Hiya,
I've followed all of the instructions to set up RCareWorld on WSL, and the only issue is that when running the test scenes, the port opens but opens in my browser instead with a loading screen instead of Unity. Does anyone have any understanding of where I could be going wrong?

**Comment by YoruCathy** [comment]
Hello @KneeShe-2004 , make sure you are on `phy-robo-care` branch. You can checkout to this branch using `git checkout phy-robo-care`

**Reply by KneeShe-2004** [reply]
Hey, I'm on the right branch!

**Reply by KneeShe-2004** [reply, marked as answer]
I did some messing around, and I think the issue is having too many background processes open :)
